src/components/common/BtnDate/BtnDate.tsx

```diff
@@ -38,32 +41,60 @@ function BtnDate(props: BtnDateProps) {
 	const isDefaultTime = time === '마감 시간';
 
 	return (
-		<BtnDateLayout
-			isPressed={isPressed}
-			isClicked={isClicked}
-			size={size.type}
-			isDelayed={isDelayed}
-			isDisabled={isDisabled}
-			isDefaultDate={isDefaultDate}
-			isDefaultTime={isDefaultTime}
-			onMouseDown={handleMouseDown}
-			onMouseUp={handleMouseUp}
-		>
-			<BtnDateText
-				icon={<CalanderIcon isDelayed={isDelayed} />}
-				text={date}
-				isDefault={isDefaultDate}
+		<ModalLayout>
+			<BtnDateLayout
+				isPressed={isPressed}
+				isClicked={isClicked}
 				size={size.type}
-			/>
-			<LineIcon size={size.type} isDelayed={isDelayed} />
-			<BtnDateText icon={<ClockIcon isDelayed={isDelayed} />} text={time} isDefault={isDefaultTime} size={size.type} />
-			<XIcon isClicked={isClicked} size={size.type} />
-		</BtnDateLayout>
+				isDelayed={isDelayed}
+				isDisabled={isDisabled}
+				isDefaultDate={isDefaultDate}
+				isDefaultTime={isDefaultTime}
+				onMouseDown={handleMouseDown}
+				onMouseUp={handleMouseUp}
+			>
+				<BtnDateText
+					icon={<CalanderIcon isDelayed={isDelayed} />}
+					text={date}
+					isDefault={isDefaultDate}
+					size={size.type}
+				/>
+				<LineIcon size={size.type} isDelayed={isDelayed} />
+				<BtnDateText
+					icon={<ClockIcon isDelayed={isDelayed} />}
+					text={time}
+					isDefault={isDefaultTime}
+					size={size.type}
+				/>
+				<XIcon isClicked={isClicked} size={size.type} />
+			</BtnDateLayout>
+			{isClicked && (
+				<>
+					<DateCorrectionModal
+						top={MODAL.DATE_CORRECTION.TASK_MODAL.top}
+						left={MODAL.DATE_CORRECTION.TASK_MODAL.left}
+					/>
+					<ModalBackdrop onClick={handleMouseUp} />
+				</>
+			)}
+		</ModalLayout>
 	);
 }
 
 export default BtnDate;
 
+const ModalLayout = styled.div`
+	position: relative;
+`;
+
+const ModalBackdrop = styled.div`
+	position: absolute;
+	top: 0;
+	z-index: 3;
+	width: 100vw;
+	height: 100vh;
+`;
```

Comment:
```suggestion
const ModalBackdrop = styled.div`
	position: fixed;
	inset: 0;
	z-index: 3;
`;
```

정도로 간략하게 표현해볼 수도 있을 것 같습니다!
스크롤되어도 뒷배경이 꽉 차게 할 수 있는데 우리 서비스는 스크롤이 없어서.. 참고만 해주시면 감사하겠습니당